storage/pebble/accounts.go

```diff
@@ -0,0 +1,86 @@
+package pebble
+
+import (
+	"encoding/binary"
+	"errors"
+	"github.com/ethereum/go-ethereum/common"
+	gethTypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/onflow/flow-evm-gateway/storage"
+	errs "github.com/onflow/flow-evm-gateway/storage/errors"
+	"math/big"
+	"sync"
+)
+
+var _ storage.AccountIndexer = &Accounts{}
+
+type Accounts struct {
+	store *Storage
+	mux   sync.RWMutex
+	// todo LRU caching with size limit
+	nonceCache map[common.Address]uint64
```

Comment:
you should consider an LRU cache for the production version of this since a map will grow unbounded.

Reply:
How come we don't want this persisted in storage? How would the `nonceCache` get properly filled-up in case of a restart due to a version version deployed?